docs/operation/volume_workflow/storageclass_creation.rst

```diff
@@ -0,0 +1,49 @@
+StorageClass Creation
+=====================
+**MetalK8s** makes use of StorageClasses in other to provision and use
+persistent volumes.
+This section hightlights how to create a Storageclass using the **CLI** only.
+
+Volume objects require a storageclass object to function properly. To create a
+storageclass object, perform the following steps.
+
+#. Create a storage class defintion.
+
+   Save the storage class definition example file as **storageclass.yaml**.
+
+   .. code-block:: shell
+
+       apiVersion: storage.k8s.io/v1
+       kind: StorageClass
+       metadata:
+         name: <storageclass_name>
+       provisioner: kubernetes.io/no-provisioner
+       reclaimPolicy: Retain
+       volumeBindingMode: WaitForFirstConsumer
+       mountOptions:
+         - rw
+       parameters:
+         fsType: <filesystem_type>
+         mkfsOptions: '["-m", "0"]'
+
+  .. tip::
```

Comment:
I wouldn't use a `tip` for that, as it's mandatory.

I would also document what the user can customize: `mountOptions`, `fsType` and `mkfsOptions`.

e.g.:

```
You can use:
* **mountOptions** to specify how the volume should be mounted
* **fsType** to specify the filesystem to use on the volume
* **mkfsOptions** to specify how the volume should be formatted (not that the options are passed as a JSON-encoded string).

**volumeBindingMode** should always be set to **WaitForFirstConsumer** in order to have properly working **Volume**.
```